tests/contrib/django/test_django_snapshots.py

```diff
@@ -77,6 +78,10 @@ def test_middleware_trace_callable_view(client):
     assert client.get("/feed-view/").status_code == 200
 
 
+@pytest.mark.skipif(
+    sys.version_info >= (3, 10, 0),
+    reason=("func_name changed with Python 3.10 which changes the resource name." "TODO: new snapshot required."),
```

Comment:
should we just do this now? should only be a few snapshots?

Reply:
I was thinking this could be done separately later as there's already a ton going on in this PR but I don't feel strongly on the matter

Reply:
since we updated other tests to conform to 3.10 doing it here should be ok

I also don't have a strong opinion here

Reply:
Definitely understand the hesitation to add one more set of changes here and we can have a smaller PR for just the django snapshot changes. I'm surprised there aren't more breaking tests because of the change to resources.